src/lib/bootloader/autoyast_converter.rb

```diff
@@ -101,6 +101,13 @@ def import_default(data, default)
           val = data.global.public_send(key)
           next unless val
 
+          # import resume only if device exists (bsc#1187690)
+          resume = val[/(?:\s|\A)resume=(\S+)/, 1]
+          if resume && !Yast::BootStorage.staging.find_by_any_name(resume)
+            log.warn "remove resume parameter for non existing #{resume}"
+            val = val.gsub(/(?:\s|\A)resume=#{Regexp.escape(resume)}/, "")
```

Comment:
Question: is not valid to simply use the same as above here? I.e., `val.gsub(/(?:\s|\A)resume=\S+/, "")`

Reply:
yes, it can be used. I just want to protect here if there are multiple resumes, to not remove valid ones ( even if multiple resumes does not make much sense, but due to some perioud of buggy behavior some autoyast profiles, can contain it )